RecoLocalMuon/Configuration/python/RecoLocalMuon_EventContent_cff.py

```diff
@@ -5,9 +5,11 @@
 # AOD content
 RecoLocalMuonAOD = cms.PSet(
     outputCommands = cms.untracked.vstring(
+        'keep *_dt1DRecHits_*_*',
         'keep *_dt4DSegments_*_*', 
         'keep *_dt4DCosmicSegments_*_*',
-        'keep *_cscSegments_*_*', 
+        'keep *_cscSegments_*_*',
+        'keep *_csc2DRecHits_*_*',
```

Comment:
we discussed with PPD and concluded to restrict this to bParking in 120X as well.
@rappoccio